### PR TITLE
Release 2.19.0

### PR DESCRIPTION
**Pull request by jpogran**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+
+INTERNAL:
+
+ - deps: Update to Node 16 and VS Code 1.61 [904](https://github.com/hashicorp/vscode-terraform/pull/904)
+ - deps: Bump @vscode/test-electron from 2.0.1 to 2.0.3 [899](https://github.com/hashicorp/vscode-terraform/pull/899)
+ - deps: Bump jest from 27.4.6 to 27.4.7 [892](https://github.com/hashicorp/vscode-terraform/pull/892)
+ - deps: Update actions/setup-node to v2 [897](https://github.com/hashicorp/vscode-terraform/pull/897)
+ - deps: Update eslint and minimal ruleset [896](https://github.com/hashicorp/vscode-terraform/pull/896)
+ - Test VS Code Version Matrix [886](https://github.com/hashicorp/vscode-terraform/pull/886)
+ - Ignore jest config when packaging [895](https://github.com/hashicorp/vscode-terraform/pull/895)
```
Comment (radeksimko) [inline CHANGELOG.md:19]:
```suggestion
 - Deprecate terraform.languageServer.requiredVersion [#903](https://github.com/hashicorp/vscode-terraform/pull/903)

ENHANCEMENTS:

 - Update telemetry configuration documentation [#894](https://github.com/hashicorp/vscode-terraform/pull/894)

INTERNAL:

 - deps: Update to Node 16 and VS Code 1.61 [#904](https://github.com/hashicorp/vscode-terraform/pull/904)
 - deps: Bump @vscode/test-electron from 2.0.1 to 2.0.3 [#899](https://github.com/hashicorp/vscode-terraform/pull/899)
 - deps: Bump jest from 27.4.6 to 27.4.7 [#892](https://github.com/hashicorp/vscode-terraform/pull/892)
 - deps: Update actions/setup-node to v2 [#897](https://github.com/hashicorp/vscode-terraform/pull/897)
 - deps: Update eslint and minimal ruleset [#896](https://github.com/hashicorp/vscode-terraform/pull/896)
 - Test VS Code Version Matrix [#886](https://github.com/hashicorp/vscode-terraform/pull/886)
 - Ignore jest config when packaging [#895](https://github.com/hashicorp/vscode-terraform/pull/895)
```

**Review by radeksimko — APPROVED** [review]

**Comment by dbanck** [comment]
I’m in favor of not listing bumps of dev dependencies like, jest or internal house keeping things, because they provide no direct value to the user

**Comment by jpogran** [comment]
I think the internal section is for just that, internal or house keeping stuff. Changelog is for us just as much as it is for external people.

**Comment by radeksimko** [comment]
For the `INTERNAL` section - I think historically the idea behind it was to list "significant enough" changes which have significant potential to introduce some bugs or misbehaviours, even though from end-user perspective they're intended to be no-op.

For LS this would likely include the JSON-RPC library, LSP structs and similar, but _not_ something like `go-cmp` which is just a library we use to compare structs in tests.

In this particular case I would probably only include the NodeJS/VS Code requirement upgrade and omit the rest.

**Pull request merged**